### ISSUE TITLE
Panel Items tab lists launchers without name

### ISSUE THREAD

**Issue by maqp** [QubesOS/qubes-issues#6148]
**Qubes OS version**: 4.0.3

**Affected component(s) or functionality** XFCE4 panel

**Brief summary**: Panel Items tab lists launchers without name

**To Reproduce**

1. Drag one or more launchers to the XFCE4 panel
2. Open Panel settings and switch to Items tab and see multiple `Launcher` entries that are hard to place in order.

**Expected behavior**: The Items tab of panel settings displays domain and Application name, e.g. `personal: Files`

**Actual behavior**: The Items tab of panel settings displays the word `Launcher`

**Screenshots**: n/a

**Additional context**: n/a

**Solutions you've tried**: Manual sorting of launchers by counting visible launchers on panel from from left to right and counting again from top to bottom in the panel settings' list when reordering, and observing to see what changes. There must be a better way.

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**: n/a

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**: n/a

This might be an upstream bug that prevents altering the text `Launcher`, but even then it could be addressed by Qubes: When hovering the mouse over some `Launcher` word, the OS will pop up a text box containing `internal name: launcher-1` for example. This internal name is apparently defined by the directory into which the launcher is created, e.g. `/home/user/.config/xfce4/panel/launcher-1/1603591973.desktop`. Thus, by setting the launcher directory as e.g. `personal_:_files` instead of `launcher-1` would make the pop-up box content much more useful. 

A fix for the name `Launcher` is the preferred one as it makes it easier to grasp the whole picture about in what order the panel items are.

**Comment by andrewdavidwong** [comment]
> that are hard to place in order

Isn't it still easy, since you can see the launchers move around on on the panel as you move them around in the settings config?

**Comment by andrewdavidwong** [comment]
> This might be an upstream bug that prevents altering the text Launcher

I do suspect that this is an upstream issue.

> but even then it could be addressed by Qubes: When hovering the mouse over some Launcher word, the OS will pop up a text box containing internal name: launcher-1 for example.

Perhaps, but then this would be an enhancement request rather than a bug report.

**Comment by maqp** [comment]
>Isn't it still easy, since you can see the launchers move around on on the panel as you move them around in the settings config?

Just finding the launcher you want to move with 20+ launchers becomes annoying. Spotting which launcher moves when clicking the move button isn't always successful on first try, and once you know which one you've been moving, you have to move it back and then try to estimate or count the distance to the launcher you were supposed to be moving.

>Perhaps, but then this would be an enhancement request rather than a bug report.

Agreed. Is it possible to change the label or should I recreate the issue?

**Comment by andrewdavidwong** [comment]
> Just finding the launcher you want to move with 20+ launchers becomes annoying. Spotting which launcher moves when clicking the move button isn't always successful on first try, and once you know which one you've been moving, you have to move it back and then try to estimate or count the distance to the launcher you were supposed to be moving.

Ah, I can see how it becomes more difficult the more launchers you have.

> Agreed. Is it possible to change the label or should I recreate the issue?

No problem, I'll change the labels on this issue.

**Comment by marmarta** [comment]
Yeah, this is XFCE issue, and I don't think we should be fixing it (although 100% agreement it's annoying, this should be fixed by XFCE).
